bot/api/service/src/main/kotlin/Transformers.kt

```diff
@@ -70,9 +73,28 @@ private fun BotBus.toRequestContext(): RequestContext =
         applicationId,
         userId,
         botId,
-        userPreferences.toUserData()
+        userPreferences.toUserData(),
+        connectorData.metadata,
+        if (booleanProperty("tock_bot_api_actions_history_to_client_bus", false)) toActionsHistory() else null,
```

Comment:
It would be great to add a property in UserRequest like `val returnsHistory:Boolean = false` in order to write

```kotlin
private val returnsAlwaysHistory = booleanProperty("tock_bot_api_actions_history_to_client_bus", false)

if (returnsAlwaysHistory || request.returnsHistory:Boolean) toActionsHistory() else null,
```

Reply:
I don't see how to add this `request.returnsHistory` properly, without modifying BotBus or adding some extra parameters in methods to transforms the data.

Reply:
The idea is to make it available to ClientBus ?

Reply:
I tried to make it available from a story scope in TockClientBus, for exemple with a bus.returnsHistory = true.
However it did not worked so i rolled back my changes.
@vsct-jburet : if you have time to explain what you wanted to do with `request.returnsHistory`, then in what use?

Reply:
I've pushed my proposal with some other minor fixes here https://github.com/theopenconversationkit/tock/commit/619e68fd3fd1cba221698f492876e5b3d6d940d0

I plan to merge the PR tomorrow if ok for you ;)